cypress/integration/Regression/CreateModels/AllEntityTypes.spec.js

```diff
@@ -7,6 +7,23 @@ import * as models from '../../../support/Models'
 import * as entities from '../../../support/Entities'
 import * as helpers from '../../../support/Helpers'
 
+// Alternately return true or false.
+// Starts out with different value depending on the day of the year.
+// Can't do this with simple variables due to the way Cypress works.
```

Comment:
> Can't do this with simple variables due to the way Cypress works.

What did you mean by this?  Seems like a closure thing instead of Cypress thing.

Reply:
You are correct. I fixed the comment.